evap/static/ts/src/notebook.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
```

Comment:
This is just to satisfy codacy, right?

@niklasmohrin I think we don't want this in the codebase then, do we?

Reply:
yeet it